lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -40,9 +40,9 @@ class BitmapCanvas extends EngineCanvas {
   /// The size of the paint [bounds].
   ui.Size get size => _bounds.size;
 
-  /// The last paragraph style is cached to optimize the case where the style
-  /// hasn't changed.
-  ParagraphGeometricStyle? _cachedLastStyle;
+  /// The last CSS font string is cached to optimize the case where the font
+  /// styles hasn't changed.
+  String? _cachedLastCssFont = null;
```

Comment:
is this covered by any benchmark ?

Reply:
Not at the moment, no.

Reply:
@ferhatb was there something specific you wanted to benchmark? This is covered in the general text layout benchmark. If it regresses or improves, that benchmark would be affected.

This change may cause more cache hits during painting text. But that's not the main purpose of the change. The main purpose is to be able to use the same API in the new rich text painter that's coming (there's no `ParagraphGeometricStyle` in the new implementation).